seats.js

```diff
@@ -0,0 +1,25 @@
+const seatsSection = document.getElementById('seats');
+const seatsTable = seatsSection.children[0];
+const seatsTableBody = seatsTable.children[0];
+
+
+//for loop section 
+console.log('Logging our Loops:')
+ for (let i = 0; i < seatsTableBody.children.length; i++) {
+   // iterate over our table rows
+   console.log('index of Body: ' + i)
```

Comment:
please remove console logs. These should only be there for development and removed before making a PR. I am seeing this a few times